Subject: Update .gitignore from Debian packaging branch

The Debian packaging git branch contains these updates to the upstream
.gitignore file(s).  This patch is autogenerated, to provide these
updates to users of the official Debian archive view of the package.

[dgit (3.11~deb9u1) update-gitignore]
---
diff --git a/.gitignore b/.gitignore
index 2b62c96..1ffc8d3 100644
--- a/.gitignore
+++ b/.gitignore
@@ -3,3 +3,5 @@ tests/goser/ff/goser_ffjson.go
 tests/types/ff/everything_ffjson.go
 tests/number/ff/number_ffjson.go
 tests/ff_ffjson.go
+
+.pc
